src/MeetingsSDKAdapter.js

```diff
@@ -818,6 +827,69 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     return concat(getDisplayData$, sharingEvents$);
   }
 
+  /**
+   * Attempts to toggle roster to the given meeting ID.
+   * A roster toggle event is dispatched.
+   *
+   * @private
+   * @param {string} ID ID of the meeting to toggle roster
+   */
+  async handleRoster(ID) {
+    const sdkMeeting = await this.fetchMeeting(ID);
+    const showRoster = !this.meetings[ID].showRoster;
+
+    this.meetings[ID].showRoster = showRoster;
+
+    sdkMeeting.emit(EVENT_ROSTER_TOGGLE, {
+      state: showRoster
+        ? MeetingControlState.ACTIVE
+        : MeetingControlState.INACTIVE,
+    });
+  }
+
+  /**
+   * Returns an observable that emits the display data of a roster control.
+   *
+   * @private
+   * @param {string} ID ID of the meeting to toggle roster
+   * @returns {Observable.<MeetingControlDisplay>} Observable stream that emits display data of the roster control
+   */
+  rosterControl(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+    const active = {
+      ID: ROSTER_CONTROL,
+      icon: 'participant-list',
+      tooltip: 'Hide participants panel',
+      state: MeetingControlState.ACTIVE,
+      text: 'Participants',
+    };
+    const inactive = {
+      ID: ROSTER_CONTROL,
+      icon: 'participant-list',
+      tooltip: 'Show participants panel',
+      state: MeetingControlState.INACTIVE,
+      text: 'Participants',
+    };
+
+    const initialState$ = new Observable((observer) => {
```

Comment:
You can keep your implementation as-is since you're following the pattern (thanks for that!) but for future reference, I would like us to move implementations to using `BehaviorSubject`s as it seems to simplify the code